src/kubernetes.cc

```diff
@@ -791,9 +799,11 @@ void KubernetesReader::WatchMaster(
   const std::string watch_param(prefix + kWatchParam);
   const std::string endpoint(
       config_.KubernetesEndpointHost() + path + watch_param);
+  auto service = std::make_shared<boost::asio::io_service>();
```

Comment:
Just musing — should this be called, e.g., `watch_service`? Just `service` seems too generic.

Reply:
Sure, done.